spectrum-offchain/src/backlog.rs

```diff
@@ -408,7 +408,7 @@ mod tests {
 
     #[tokio::test]
     async fn should_pop_best_order() {
-        let mut backlog = setup_backlog(10, 5, 50);
+        let mut backlog = setup_backlog(10, 5, 0);
```

Comment:
Set the probability to zero, otherwise the test fails when trying to pop a suspended order that doesn't exist.